go/theatre/StatementPrinter_test.go

```diff
@@ -0,0 +1,74 @@
+package theatre_test
+
+import (
+	"encoding/json"
+	"testing"
+
+	"github.com/emilybache/Theatrical-Players-Refactoring-Kata/go/theatre"
```

Comment:
I'm not familiar with Go, but this looks to me like it would put a dependency on github rather than the local copy of this file. Then if you refactored locally the test would still pass since it executes the remote file. Is that right or did I just misunderstand how Go works?